lib/ansible/modules/user.py

```diff
@@ -3260,6 +3283,10 @@ def main():
                 result['ssh_fingerprint'] = err.strip()
             result['ssh_key_file'] = user.get_ssh_key_path()
             result['ssh_public_key'] = user.get_ssh_public_key()
+            if user.ssh_passphrase is None and user.ssh_key_rounds is not None:
+                warning_msg = 'Option \'ssh_key_rounds\' of module \'ansible.builtin.user\' is useless'\
+                    ' without \'ssh_passphrase\' specified. See module documentation.'
+                result['warnings'].append(warning_msg)
```

Comment:
use `module.warn` instead

Reply:
I changed this. Thanks for the hint. What is stillt to do to get this PR merged?

Reply:
Not much, the main consideration at this point is 'overload' of the user module, many of these options seem to belong on a 'keygen' module instead, bringing this up to core to make a final decision.